File: db4o.net/Db4objects.Db4o.Tests/Db4objects.Db4o.Tests/Common/Reflect/Custom/PersistentEntryTemplate.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

namespace Db4objects.Db4o.Tests.Common.Reflect.Custom
{
	public class PersistentEntryTemplate
	{
		public string className;

		public string[] fieldNames;

		public object[] fieldValues;

		public PersistentEntryTemplate(string className, string[] fieldNames, object[] fieldValues
			)
		{
			this.className = className;
			this.fieldNames = fieldNames;
			this.fieldValues = fieldValues;
		}
	}
}

File: db4oj.tests/src/com/db4o/db4ounit/common/reflect/custom/PersistentEntryTemplate.java
package com.db4o.db4ounit.common.reflect.custom;

public class PersistentEntryTemplate {

	public String className;

	public String[] fieldNames;

	public Object[] fieldValues;

	public PersistentEntryTemplate(String className, String[] fieldNames, Object[] fieldValues)
	{
		this.className = className;
		this.fieldNames = fieldNames;
		this.fieldValues = fieldValues;
	}
}
